app/assets/javascripts/no_sunet.js

```diff
@@ -0,0 +1,43 @@
+$(document).on('click', '#no-sunet-button', function(){
+  setTimeout(function(){
```

Comment:
This enables the Library ID field to get the focus when this script runs, otherwise when the no-sunet button is clicked that button still has the focus and the the library_id field can't get it.